src/app/components/Figure/Caption/__snapshots__/index.test.jsx.snap

```diff
@@ -29,6 +30,12 @@ exports[`Caption should render correctly 1`] = `
   }
 }
 
+@media (min-width:37.5em) {
+  .c0 {
+    padding: 0.5rem 1rem;
+  }
+}
+
```

Comment:
Note that by separating the Typography CSS out and still having a media query in the component for other styles, we get verbose CSS output. 

```
@media (min-width:37.5em) {
  .c0 {
    font-size: 0.875em;
  }
}

@media (min-width:37.5em) {
  .c0 {
    padding: 0.5rem 1rem;
  }
}
```
Instead of 
```
@media (min-width:37.5em) {
  .c0 {
    font-size: 0.875em;
    padding: 0.5rem 1rem;
  }
}
```

Separately to this PR, we should consider optimisation of styles outputted by `styled-components`.

Reply:
This is a good point. It's a question of DX vs CSS optimisation vs styled components CSS output. 

In the isolated example you've described having the typography split out ends up with overly verbose CSS. However given an example with multiple components on a single pages using the same typography groups the opposite effect happens as CSS is reduced. Take the example below:
```
// HTML
<p class="123">some text.</p>
<figure>
  <img ...>
  <figcaption class="123 456">some caption.</figcaption>
</figure>


// CSS
.123 {
  * typography definition here *
};

.456 {
  * similar media query here *
};
```

Giving the typography is split out and extended within the styled components <p> and <figcaption> then the styles are both put under the class "123".

Whereas if the typography was not split out styled components would render the typography css for each different component on the page and therefore there would be overly verbose css on a page level.

Agreed we need a PR to analyse and ideally optimise the output from styled components.

Reply:
This has been captured in https://github.com/bbc/simorgh/issues/550